speech/cloud-client/src/main/java/com/example/speech/Recognize.java

```diff
@@ -46,6 +46,8 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
```

Comment:
Let's reset this file back to it's original format. (Since we pulled the sample out)